### PR TITLE
Add aws_flow_logs_s3_buckets module

### PR DESCRIPTION
**Pull request by rlenglet**
Add new `aws_flow_logs_s3_buckets` submodule.

In `aws_account`, replace the usage of `random_uuid` with `random_password` to increase the strength of the role external ID.

Improve formatting and structure of examples.
Delete empty `outputs.tf` files.
Fix the validation of variables in `aws_account` example.

Set version to v1.2.0.

### CODE REVIEW

File: modules/aws_flow_logs_s3_buckets/variables.tf [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    condition     = length(var.role_id) > 1
+    error_message = "The role_id value must not be empty."
+  }
+}
+
+variable "s3_bucket_arns" {
+  description = "The set of ARNs of AWS S3 buckets containing flow logs."
+  type        = set(string)
+  validation {
+    condition     = length(var.s3_bucket_arns) > 1
```
Comment (shireesh-illumio) [inline modules/aws_flow_logs_s3_buckets/variables.tf:24]: this should be "> 0"

**Review by shireesh-illumio — CHANGES_REQUESTED** [review]
also can we modify the random_uuid to random_password as in #10 
makes module secure.

File: modules/aws_flow_logs_s3_buckets/variables.tf (hunk 1 above)
Comment (rlenglet) [reply modules/aws_flow_logs_s3_buckets/variables.tf]: Good catch. Fixed.

**Comment by rlenglet** [comment]
> also can we modify the random_uuid to random_password as in https://github.com/illumio/terraform-illumio-cloudsecure/pull/10
> makes module secure.

Good idea. Done!

**Review by shireesh-illumio — APPROVED** [review]
LGTM. we can ship it. thank you

**Pull request merged**